deltachat-jsonrpc/src/api/types/message.rs

```diff
@@ -708,3 +708,42 @@ impl From<deltachat::ephemeral::Timer> for EphemeralTimer {
         }
     }
 }
+
+#[derive(Deserialize, TypeDef, schemars::JsonSchema)]
+#[serde(rename_all = "camelCase")]
+pub struct LateFilingMediaSize {
+    // The new width to store in the message object. None if you don't want to change the width.
+    pub width: Option<u32>,
```

Comment:
Maybe replace these with `pub wh: Option<(u32, u32)>,`? Because if `width == None`, the height isn't updated by the core too, the same otherwise. Or at least document this in the comments.